docs/configuration.rst

```diff
@@ -5,8 +5,33 @@ Configuration
    :caption: application.cfg
 
    # Sample application.cfg file
+   import os
+
+   # Database paths
+   CELERY_BROKER_URL = 'redis://redis'
+   SQLALCHEMY_DATABASE_URI = 'postgresql://postgres:mysecretpassword@postgres/postgres'
```

Comment:
Same for SQLALCHEMY_DATABASE_URI: the developer instructions involve setting up the Postgres database in a way that works by default.